1-exercises/A-undefined/exercise.js

```diff
@@ -12,25 +12,28 @@
 // Example 1
 let a;
 console.log(a);
-
+// this is undefined because it's null , ( contains no value )
```

Comment:
Actually, it's `undefined`, not `null`. For something to be `null` we have to explicitly mark it as such, like this:

```js
let a = null;
```

You would use `null` when you want to intentionally mark something as empty/missing, e.g. in this list, the singer Cher has no last name - this isn't a mistake, it's just a fact of her name:

```js
const artists = [
  {
    firstName: 'Ravi',
    lastName: 'Shankar',
    numberOfSongs: 93
  },
  {
    firstName: 'Cher',
    lastName: null,
    numberOfSongs: 72    
  },
  {
    firstName: 'Miles',
    lastName: 'Davis',
    numberOfSongs: 89
  }
]
```

Reply:
Thanks Tom for this clear elaborated explaination. :-)